client/web/compose/src/components/Public/Record/BulkEdit/index.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <div>
+    <b-button
+      :title="$t('recordList.bulkRecord.title')"
+      variant="outline-light"
+      class="text-primary border-0 inline float-left"
+      size="sm"
+      @click="showModal=true"
+    >
+      <font-awesome-icon
+        :icon="['far', 'edit']"
+      />
+    </b-button>
+
+    <b-modal
+      :visible="showModal"
+      size="md"
+      :title="$t('recordList.bulkRecord.title')"
+      hide-footer
+      body-class="p-0"
+      centered
+      @hide="onModalHide"
+    >
+      <b-card class="pt-0">
+        <div
+          v-if="fields.length"
+          class="mb-3 pb-3"
+        >
+          <b-row
+            v-for="(field, index) in fields"
+            :key="index"
+            class="mb-4"
+            align-v="center"
+          >
+            <b-col
+              v-if="field"
+              cols="12"
+              lg="12"
+            >
+              <field-editor
+                v-bind="mock"
+                :field="field"
+                class="field-editor mb-0"
+              />
+            </b-col>
+          </b-row>
+        </div>
+
+        <b-form-group>
+          <b-input-group>
+            <vue-select
+              v-model="currentField"
+              :placeholder="$t('recordList.bulkRecord.searchFields')"
+              :get-option-label="getOptionLabel"
+              :options="filterFieldOptions()"
+              class="bg-white position-relative"
+            />
+
+            <b-input-group-append>
+              <b-button
+                variant="primary"
+                class="px-4"
+                @click="addField"
```

Comment:
Instead of having and add, just add the field on select, would also remove the need for input group